### PR TITLE
Eventhub: Added Namespace OperationsID API

### PR DESCRIPTION
**Pull request by v-Ajnava**
### Latest improvements:
<i>MSFT employees can try out our new experience at <b>[OpenAPI Hub](https://aka.ms/openapiportal) </b> - one location for using our validation tools and finding your workflow. 
</i><br>
### Contribution checklist:
- [X] I have reviewed the [documentation](https://github.com/Azure/adx-documentation-pr/wiki/Overall-basic-flow) for the workflow.
- [X] [Validation tools](https://github.com/Azure/azure-rest-api-specs/blob/master/documentation/swagger-checklist.md#validation-tools-for-swagger-checklist) were run on swagger spec(s) and have all been fixed in this PR.
- [X] The [OpenAPI Hub](https://aka.ms/openapiportal) was used for checking validation status and next steps.
### ARM API Review Checklist
- [X] Service team MUST add the "WaitForARMFeedback" label if the management plane API changes fall into one of the below categories. 
- adding/removing APIs.
- adding/removing properties.
- adding/removing API-version. 
- adding a new service in Azure.

Failure to comply may result in delays for manifest application. Note this does not apply to data plane APIs.
- [X] If you are blocked on ARM review and want to get the PR merged urgently, please get the ARM oncall for reviews (RP Manifest Approvers team under Azure Resource Manager service) from IcM and reach out to them. 
Please follow the link to find more details on [API review process](https://armwiki.azurewebsites.net/rp_onboarding/ResourceProviderOnboardingAPIRevieworkflow.html).

### CODE REVIEW

File: specification/eventhub/resource-manager/Microsoft.EventHub/stable/2017-04-01/EventHub.json [hunk 1]
```diff
@@ -2074,6 +2074,54 @@
           "nextLinkName": "nextLink"
         }
       }
+    },
+    "/subscriptions/{subscriptionId}/providers/Microsoft.EventHub/locations/{locationName}/namespaceOperations/{operationResultId}": {
```
Comment (ryansbenson) [inline specification/eventhub/resource-manager/Microsoft.EventHub/stable/2017-04-01/EventHub.json]: It looks like you're trying to expose the operation status of an async api. There is a pattern in ARM for this, you'd expose:

/subscriptions/{subscriptionId}/providers/Microsoft.EventHub/locations/{locationName}/operationsStatus
https://github.com/Azure/azure-resource-manager-rpc/blob/master/v1.0/Addendum.md#operation-resource-format

**Comment by yungezz** [comment]
hi @v-Ajnava could you pls address review comments? thanks.

**Comment by yungezz** [comment]
Hi @v-Ajnava any update?

**Comment by v-Ajnava** [comment]
Hi @yungezz, I am discussing with team here and will update the PR.

**Comment by yungezz** [comment]
hi @v-Ajnava just kindly ping, any update or plan?

**Comment by yungezz** [comment]
ping for update

**Comment by yungezz** [comment]
ping for update

**Comment by yungezz** [comment]
hi @v-Ajnava, are you still on this PR? Is it ok to close it now if not ready for reviewing? thanks.

**Comment by v-Ajnava** [comment]
@yungezz apologies for delay in response, was on some other task. will reply to the comments.

File: specification/eventhub/resource-manager/Microsoft.EventHub/stable/2017-04-01/EventHub.json (hunk 1 above)
Comment (v-Ajnava) [reply specification/eventhub/resource-manager/Microsoft.EventHub/stable/2017-04-01/EventHub.json]: @rrama-Microsoft can you please take look here

**Comment by v-Ajnava** [comment]
@yungezz and @ryansbenson developer is working on implementing the comments. 

let us know you views on keeping the PR open or closing it.

**Comment by yungezz** [comment]
hi @v-Ajnava as long as the PR is valid and actively been worked on, it's ok to keep it open.

**Comment by yungezz** [comment]
Any ETA so far?

**Comment by yungezz** [comment]
hi @v-Ajnava since no update in several months, is it ok to close the PR for now? thanks

**Comment by ravbhatnagar** [comment]
no response from PR owner for a long time. removing ARM review labels.

**Comment by yungezz** [comment]
hi @v-Ajnava, will close the PR for now since no activity since mid of Jan. It's out sync with master branch already. Pls feel free to re-open or create new PR and assign me for reviewing. thanks.

**Pull request closed without merging**